### ISSUE TITLE
Add support for ctags file

### ISSUE THREAD

**Issue by daedroza** [gfanto/fzf-lsp.nvim#28]
Hello, would it be possible to add support for tags file without using neovim's lsp?

**Comment by gfanto** [comment]
Yes it can be added and I really like the idea, it is a bit out of the scope of this plugin as it is not related to lsp, but as I have all the necessary bits I can think of adding it.
I think I'll add it, but I don't think I have time to add it this week, I'll let you know when I roll out the feature.

**Comment by daedroza** [comment]
Hi @gfanto ,

Any update for this issue?
